### ISSUE TITLE
Crash on polar plot display, main branch, python 3.10

### ISSUE THREAD

**Issue by arkhnchul** [NanoVNA-Saver/nanovna-saver#528]
the app crashes while trying to draw polar plot with python 3.10

```
Traceback (most recent call last):
  File "/tmp/nanovna-saver/NanoVNASaver/Charts/Square.py", line 50, in paintEvent
    self.drawChart(qp)
  File "/tmp/nanovna-saver/NanoVNASaver/Charts/Polar.py", line 49, in drawChart
    qp.drawLine(center_x + width_45, center_y + height_45,
TypeError: arguments did not match any overloaded call:
  drawLine(self, QLineF): argument 1 has unexpected type 'float'
  drawLine(self, QLine): argument 1 has unexpected type 'float'
  drawLine(self, int, int, int, int): argument 1 has unexpected type 'float'
  drawLine(self, QPoint, QPoint): argument 1 has unexpected type 'float'
  drawLine(self, Union[QPointF, QPoint], Union[QPointF, QPoint]): argument 1 has unexpected type 'float'
```

there are two variables still werent casted to integers at initialisation, width_45 and height_45, lines 33-34 of NanoVNASaver/Charts/Polar.py.

**Comment by cacheflood** [comment]
Same issue on NanoVNASaver 0.5.0

Traceback (most recent call last):
  File "/home/jani/.local/lib/python3.10/site-packages/NanoVNASaver/Charts/Square.py", line 50, in paintEvent
    self.drawChart(qp)
  File "/home/jani/.local/lib/python3.10/site-packages/NanoVNASaver/Charts/Polar.py", line 49, in drawChart
    qp.drawLine(center_x + width_45, center_y + height_45,
TypeError: arguments did not match any overloaded call:
  drawLine(self, QLineF): argument 1 has unexpected type 'float'
  drawLine(self, QLine): argument 1 has unexpected type 'float'
  drawLine(self, int, int, int, int): argument 1 has unexpected type 'float'
  drawLine(self, QPoint, QPoint): argument 1 has unexpected type 'float'
  drawLine(self, Union[QPointF, QPoint], Union[QPointF, QPoint]): argument 1 has unexpected type 'float'

**Comment by zarath** [comment]
Fixed in related issues #539 and #541
